PocketKit/Sources/PocketKit/Premium/PremiumUpgradeView.swift

```diff
@@ -175,10 +176,28 @@ Refunds are not available for unused portions of a subscription.
 }
 
 private struct PremiumTermsView: View {
+    @State var showSafari = false
+    @State var urlString = "https://getpocket.com/en/privacy/"
+    // TODO: URLs need to be localized
+    let privacyPolicyUrlString = "https://getpocket.com/en/privacy/"
+    let toSUrlString = "https://getpocket.com/en/tos/"
```

Comment:
can we set these as `private`?

Reply:
@timc-mozilla lets remove the `/en/` portion of the URL. That way if we ever localize these, the user gets that version automatically. Then you can remove the TODO you have.

Reply:
gotcha gotcha. thanks